src/main/java/internetshop/controllers/CompleteOrderController.java

```diff
@@ -0,0 +1,31 @@
+package internetshop.controllers;
```

Comment:
```suggestion
package internetshop.controller;
```
переджі потрібно називати в однині